docs/OCI_COMPLIANCE.md

```diff
@@ -0,0 +1,256 @@
+
+**Remoteproc Runtime**: **No security features implemented**.
+
+**Rationale**:
+
+- Firmware security is processor-specific (MPU, TrustZone, etc.)
+- No syscalls to filter - firmware doesn't use Linux syscalls
+- Security boundary is the hardware separation between processors
+- Host security doesn't apply to auxiliary processor execution
+
+**Impact**: Firmware trustworthiness must be ensured through:
+
+- Container image signing and verification (supply chain security)
+- Processor-level security features (Secure Boot, memory protection)
+- Hardware isolation mechanisms
+
+### 9. Additional Operations
+
+**Standard OCI**: Optional but common operations ([OCI Runtime Spec - Operations](https://github.com/opencontainers/runtime-spec/blob/main/runtime.md#operations)).
+
+**Remoteproc Runtime**: **None of these operations are supported**.
+
+**Rationale**:
+
+- **exec**: No shell or process model on auxiliary processor
+- **pause/resume**: Remoteproc framework has `suspended` state, but not exposed by runtime
+- **checkpoint/restore**: Processor state is hardware-specific, not portable
+- **update**: No runtime-modifiable parameters
+
+### 10. Device Access
+
+**Standard OCI**: Device management for passing `/dev` devices into containers ([OCI Config Spec - Linux Devices](https://github.com/opencontainers/runtime-spec/blob/main/config-linux.md#devices)).
+
+**Remoteproc Runtime**: **No device access**.
+
+**Rationale**:
+
+- Firmware runs on a separate processor with no access to Linux `/dev` devices
+- The proxy process interacts with `/sys/class/remoteproc/remoteprocN/` (sysfs, not device nodes)
+- Auxiliary processor peripherals are hardware-mapped, not Linux devices
+- Runtime needs privileges to write sysfs files, but this is a deployment requirement, not OCI device passthrough
+
+**Impact**: Firmware accesses hardware peripherals directly through processor-specific memory-mapped I/O, not through Linux device nodes.
+
+### 11. Signal Handling
+
+**Standard OCI**: Container process receives signals directly.
+
+**Remoteproc Runtime**: **Proxy-mediated signal handling**:
+
+- SIGUSR1: Start signal (transitions proxy from phase 1 to phase 2)
+- SIGTERM/SIGINT: Graceful stop (proxy stops processor via sysfs)
+- SIGKILL: Force termination (kills proxy, processor may remain running)
+
+The firmware itself cannot receive signals - it runs on a separate processor without signal infrastructure.
+
+**Rationale**: Signals control the lifecycle management proxy, not the firmware. The firmware is controlled by writing to sysfs (`state` file).
+
+### 12. Single Container per Processor Limitation
+
+**Standard OCI**: Runtimes can create and run multiple independent containers simultaneously.
+
+**Remoteproc Runtime**: Can only run **one container at a time per remote processor**. Each auxiliary processor can execute only one firmware image. The previous container must be stopped before a new one can be started.
```

Comment:
```suggestion
**Remoteproc Runtime**: Can only run **one container at a time per remote processor**. Each auxiliary processor can execute only one firmware image. Any firmware running on the target processor, be it a container from our runtime or otherwise, must be stopped before a new container can be started.
```